src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Colors.values().length);
+
+    public String getRandomColor() {
+        return Colors.values()[index].name();
+    }
```

Comment:
You should generate the random index inside the `getRandomColor` method to ensure that a new random color is provided on each method call. Currently, the `index` is generated once when the `ColorSupplier` is instantiated, which means it will always return the same color.